test/jdk/java/io/File/GetXSpace.java

```diff
@@ -238,7 +238,12 @@ private static void compare(Space s) {
             // value when the number of blocks is odd.
             if (!Platform.isOSX() || blockSize != 512 || numBlocks % 2 == 0
                 || ts - s.total() != 512) {
-                fail(s.name(), s.total(), "!=", ts);
+                // If Windows user quotas are in effect, then the total space
+                // reported by df might be less that the actual total space in
+                // the store, but the free space should be the same.
+                if (!(Platform.isWindows() && ts > s.total && fs == s.free())) {
```

Comment:
Unsure of the conditional here, in particular whether `fs == s.free()` is needed. Also, when failure is suppressed here, an informational print might be useful.

Reply:
Changed the conditional in 991dc9a6bf86e3ec54fd1c244c655a86506a697d.